src/Modules/CalcOffence.lua

```diff
@@ -262,6 +276,7 @@ end
 function calcSkillCooldown(skillModList, skillCfg, skillData)
 	local cooldownOverride = skillModList:Override(skillCfg, "CooldownRecovery")
 	local cooldown = cooldownOverride or (skillData.cooldown + skillModList:Sum("BASE", skillCfg, "CooldownRecovery")) / m_max(0, calcLib.mod(skillModList, skillCfg, "CooldownRecovery"))
+	-- TODO: Do not round to tick rate for skills that store multiple uses!
```

Comment:
I don't know where to fetch this data. Many skills seem to get a property for multiple uses upon extraction, but Doom Blast does not. At least not in sup_int.lua.

Maybe it is possible to fix that somehow, so we can use that property? 

Right now, the cooldown displayed for Doom Blast still takes tick rate into account, but the DPS calculations do not.

Reply:
I think the number of cooldown stat that can be found in multiple spots e.g. warcries is manually added I am not sure if that data is supplied. You can add it yourself if you like and use it. Second wind for example is implemented but doesn't really work on much stuff. Multi cooldown traps and flame dash for example aren't implemented.